test/testing/test_utils.py

```diff
@@ -247,3 +248,13 @@ def wait_for_event(self):
     with eventlet.Timeout(1):
         gt.wait()
     assert container._worker_pool.free() == max_workers
+
+
+def test_rabbit_connectipdateon_refused_error():
+
+    bad_port_uri = 'http://guest:guest@localhost:11111'
```

Comment:
do we need to worry about someone having this port open? we could use the trick from `test/web/conftest.py` to ask the os for a free port, and then close it and use that?

or maybe we could use one of the low, [well known](https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers#Well-known_ports) but unassigned ports, e.g. `4`